### PR TITLE
BUG: fix isin for server

### PR DESCRIPTION
**Pull request by llllllllll**
The frozenset couldn't be serialized. This also makes deserialization of the datetime give a timestamp.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -76,6 +76,9 @@ install:
   # Install coveralls
   - pip install coveralls
 
+  # Install pyyaml
+  - pip install PyYAML
```
Comment (cpcloud) [inline .travis.yml]: remove this line ... i just merged 082d96a which adds this to travis

Comment (llllllllll) [inline .travis.yml]: cool, btw, have you given any thought to the 2.6 compat question?

**Comment by llllllllll** [comment]
@cpcloud merging on pass.

File: blaze/utils.py [hunk 2]
```diff
@@ -157,7 +160,74 @@ def listpack(x):
...
+    >>> object_hook.register('frozenset', frozenset)
+    """
+    if len(obj) != 1:
+        return obj
+
+    key, = obj.keys()
+    if not key.startswith('__!'):
+        return obj
+
+    return object_hook._converters[key[3:]](obj[key])
```
Comment (cpcloud) [inline blaze/utils.py]: can you write this as `key[len('__!'):]`? IMO this clarifies what you're doing here

Comment (llllllllll) [inline blaze/utils.py]: sure

**Comment by llllllllll** [comment]
This was not the PR I meant to merge I saw the comment in my email about the len pr, Sorry about this!

**Comment by llllllllll** [comment]
Should I revert or just push the len change?

**Comment by cpcloud** [comment]
Just push the `len` change

**Comment by llllllllll** [comment]
Okay. Sorry about that.

**Pull request merged**